code/modules/meteors/meteor_spawning.dm

```diff
@@ -31,22 +31,22 @@
 	var/new_meteor = pick_weight(meteor_types)
 	new new_meteor(picked_start, picked_goal)
 
-/proc/spaceDebrisStartLoc(start_side, Z)
+/proc/spaceDebrisStartLoc(start_side, Z, distance_from_edge)
```

Comment:
```suggestion
/proc/spaceDebrisStartLoc(start_side, Z, distance_from_edge = 0)
```
not critical but cleaner this way, since other procs also use this one